docs/for-devs/get-started/lab-10.md

```diff
@@ -77,7 +73,9 @@ spec:
 
 - Commit Changes
 
-Now go back to the Argo CD application and click on the `team<name>` application. You can see that all the Kubernetes resources have been created.
+Now go back to the Argo CD application and click on the `team<name>` application.
+
+Note that the Argo CD application is not configured to automatically SYNC. So click `SYNC` and then `REFRESH`.
```

Comment:
Really? Then we need to change this setting

Reply:
there is no sync policy here: https://github.com/redkubes/otomi-core/blob/main/charts/team-ns/templates/argocd/argocd-application.yaml